Tools/CISupport/ews-build/config.json

```diff
@@ -346,6 +346,13 @@
       "type": "Try_Userpass", "name": "try_commitqueue", "port": 5557,
       "builderNames": ["Commit-Queue"]
     },
+    {
+      "type": "AnyBranchScheduler", "name": "pullrequest",
```

Comment:
how is this scheduler triggered?

Reply:
GitHub hooks generate by pull-requests have this:
`-H "X-GitHub-Event: pull_request`
in their header. My understanding is that upon receiving this hook, buildbot triggers the scheduler named `pull request`